backend/src/config.js

```diff
@@ -0,0 +1,5 @@
+export default {
+    MONGODB_URL: process.env.MONGODB_URL || 'mongodb://localhost/mygym',
+    JWT_SECRET: process.env.JWT_SECRET || 'somethingsecret', 
+    STRIPE_SK: 'sk_test_51JiCwaBQcQnzL9cb9ImcFc4aGZK9TmkfGp6tdbTP7JLQabIZEuHa2xnpcIuAqTOxWDodWRbBvuNQRknxjAyXnLMs00xOnZqz0k'
```

Comment:
Move to env variables. Put in your .env file but don't check that file in. Gitignore it.